web/src/stores.ts

```diff
@@ -61,7 +61,7 @@ export type Mode =
 export let map: Writable<Map | null> = writable(null);
 
 // The exact key in local storage
-export let projectName: Writable<string> = writable("");
+export let currentProjectKey: Writable<string> = writable("");
```

Comment:
This was just a rename to make it more obvious that it's not *only* the project name.